docs/android-setup.md

```diff
@@ -0,0 +1,93 @@
+# Installation
+
+In some cases, simply doing `react-native link` will correctly set up the tracker on Android.
```

Comment:
> some cases

Is it still believed that `react-native link` often won't work?  I don't know if I've ever seen this _not_ work.

Reply:
This line is verbatim from the old Android section, I just moved it into this new file. Do you think it needs a change?